### Set the backend chess engine difficulty.
#
# The difficulty scale is a multiplier to the chess engine's search depth.

float32 difficulty  # scale from 1.0 - 10.0 with 1.0 being the easiest
---
int8 rc   # 0 = ok, <0 = error
